geoprocessing/test/planning-units-processor.e2e-spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Job } from 'bullmq';
+import defaultExport from '../src/modules/planning-units/planning-units.job';
+
+import { E2E_CONFIG } from './e2e.config';
+
+function delay(ms: number) {
+  return new Promise(function (resolve) {
+    return setTimeout(resolve, ms);
+  });
+}
+
+describe('planning units jobs (e2e)', () => {
+  jest.setTimeout(2 * 1000);
+  it('executes the child job processor with mock data', async () => {
+    const createPlanningUnitsDTO: Partial<Job> = {
```

Comment:
this test prevents the geoprocessing service to build for me.

without full context, I got this to build with a few type narrowings - see my suggestions about this in planning-units.job.ts and in the e2e config for geoprocessing

Reply:
ok i think this is solved now